includes/Data/Plugins.php

```diff
@@ -113,6 +113,11 @@ final class Plugins {
 			'url'      => 'https://hiive.cloud/workers/plugin-downloads/wonder-cart',
 			'path'     => 'wonder-cart/init.php',
 		),
+		'nfd_slug_yith_stripe_payments_for_woocommerce' => array(
+			'approved' => true,
+			'url' => 'https://downloads.yithemes.com/?apiRequest=download_extended&package=yith-stripe-payments-for-woocommerce',
```

Comment:
Let's move this to a Cloudflare URL, not sure if we want to be exposing this as is.

Reply:
https://hiive.cloud/workers/plugin-downloads/yith-stripe-payments-for-woocommerce

Reply:
updated the url